magefiles/installation/install.go

```diff
@@ -149,6 +187,159 @@ func (i *InstallAppStudio) cloneInfraDeployments() (*git.Remote, error) {
 	return repo.CreateRemote(&config.RemoteConfig{Name: i.LocalForkName, URLs: []string{fmt.Sprintf("https://github.com/%s/infra-deployments.git", i.LocalGithubForkOrganization)}})
 }
 
+func MergePRInRemote(branch string, forkOrganization string, repoPath string) error {
+	if forkOrganization == "" {
+		forkOrganization = "redhat-appstudio"
+	}
+
+	if branch == "" {
+		klog.Fatal("The branch for upgrade is empty!")
+	}
+
+	cmd, err := exec.Command("git", "-C", repoPath, "branch").CombinedOutput()
+	if err != nil {
+		klog.Fatal(err)
+	}
+	fmt.Printf("output repo branches: %s\n", cmd)
```

Comment:
This is just debug info which will be removed later? I can't see it being used later on.

Reply:
Removed